test/tests/parallel/set1/fetchmail.bats

```diff
@@ -60,6 +60,13 @@ function teardown_file() {
   _should_not_have_in_config 'pop3.third-party.test. '   /etc/fetchmailrc.d/fetchmail-2.rc
 }
 
+@test "(ENV FETCHMAIL_PARALLEL=1) should detect info line in fetchmail-x.rc" {
+  export CONTAINER_NAME=${CONTAINER2_NAME}
+
+  _should_not_have_in_config      'INFO'                      /etc/fetchmailrc.d/fetchmail-1.rc
+  _should_note_have_in_config     'INFO'                      /etc/fetchmailrc.d/fetchmail-2.rc
+}
```

Comment:
- There is a typo on the 2nd `_should` line, `note` => `not`.
- The test name implies it should detect an info line, while the assertions are that it should not find such a line 😅

```suggestion
@test "(ENV FETCHMAIL_PARALLEL=1) should not have logs emitted into fetchmail-x.rc" {
  export CONTAINER_NAME=${CONTAINER2_NAME}

  _should_not_have_in_config      'INFO'                      /etc/fetchmailrc.d/fetchmail-1.rc
  _should_not_have_in_config      'INFO'                      /etc/fetchmailrc.d/fetchmail-2.rc
}
```